curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 6676a8b01e56ec1a1e07c202
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+Notice that you now have a white square with thick borders. The borders have diagonal edges which can be used to create triangles.
+
+Within the same class selector adjust `height` and `width` to `0`. Set the left, right and top border to `transparent`.
+
+# --hints--
+
+
+You should have a `.cat-right-ear` selector.
+
+```js
+assert(new __helpers.CSSHelp(document)?.getStyle('.cat-right-ear'))
+```
+
+Your `.cat-right-ear` selector should have a `height` property set to `0`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.height == '0px')
+```
+
+Your `.cat-right-ear` selector should have a `width` property set to `0`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.width === '0px')
+```
+
+Your `.cat-right-ear` selector should have a `background-color` property set to `white`. Don't forget to add a semicolon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.backgroundColor === 'white')
+```
+
+Your `.cat-right-ear` selector should have a `border-left` property set to `35px solid transparent`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderLeft === '35px solid transparent')
+```
+
+Your `.cat-right-ear` selector should have a `border-right` property set to `35px solid transparent`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderRight === '35px solid transparent')
+```
+
+Your `.cat-right-ear` selector should have a `border-top` property set to `35px solid transparent`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderTop === '35px solid transparent')
+```
+
+
+Your `.cat-right-ear` selector should have a `border-bottom` property set to `35px solid red`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderBottom === '35px solid red')
+```
+
```

Comment:
````suggestion

Your `.cat-right-ear` selector should not have a `background-color` property.

```js
assert(!new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.backgroundColor)
```

````

And test for it.